docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -6,6 +6,7 @@ This page explains how to configure the Kubernetes Security Posture Management (
 [sidebar]
 --
 * The KSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, ensure you have the `read` privilege to the following elasticsearch indices: \n- `logs-cloud_security_posture.findings_latest-*`\n- `logs-cloud_security_posture.scores-*`
```

Comment:
I think it's also worth adding logs-cloud_security_posture.findings so they can view that index in discover (and for future history features, debugging and so on).